websites/site-template/gatsby-config.mjs

```diff
@@ -13,21 +14,34 @@ const config = {
   siteMetadata: {
     title: 'CHANGE-ME',
     description: 'CHANGE-ME',
+    products: ['smokeTest'],
+    contentType: 'test',
     breadcrumbs: 'CHANGE-ME',
-    betaLink: '',
-    excludeFromSearchIndex: isProd,
-    allowWideContentLayout: true,
-    products: ['CHANGE-ME'],
-    contentType: 'CHANGE-ME',
+    betaLink: '/beta',
+    excludeFromSearchIndex: isProd && !shouldEnableSearch,
   },
   plugins: [
     ...configureThemeWithAddOns({
```

Comment:
If I understand correctly you're re-writing the gatsby-config.mjs file at each microsite creation, so why even bother in adjusting this file? why not leave it how it was?

Reply:
Because the site template was outdated for a while already. I just adjusted the config to our current setup (like we do it in the other microsites).